docs/src/app_dev/app_val_tokenomics/feat_app_act_marker_nomics.rst

```diff
@@ -0,0 +1,83 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+..
+   SPDX-License-Identifier: Apache-2.0
+
+.. _feat_app_act_marker_nomics:
+
+Featured Application Activity Marker
+=====================================
+
+Featured application activity markers
+(`FeaturedAppActivityMarker <https://docs.dev.sync.global/app_dev/api/splice-api-featured-app-v1/Splice-Api-FeaturedAppRightV1.html#splice-api-featuredapprightv1>`__)
+are created in a transaction for activity on the network that adds value
+but does not involve a CC Transfer (i.e., transfers a RWA like a USDC).
+A ``FeaturedAppActivityMarker`` is immediately converted into an
+``AppRewardCoupon`` by the automation run by the SuperValidators and the
+``AppRewardCoupon`` is minted just like in a :ref:`cc_xfer_nomics`.
+Please review the application activity marker
+`CIP <https://github.com/global-synchronizer-foundation/cips/blob/main/cip-0047/cip-0047.md>`__
+for details. A summary follows.
+
+There are two prerequisites for an application to create a
+``FeaturedAppActivityMarker``. The first is to become an approved featured
+application which was described in the
+:ref:`app_val_tokenomics`
+section. The second is to update the application code:
+
+   1. Find the fully qualified interface ID for
+   `FeaturedAppRight <https://docs.dev.sync.global/app_dev/api/splice-api-featured-app-v1/Splice-Api-FeaturedAppRightV1.html#type-splice-api-featuredapprightv1-featuredappright-34177>`__
+   which is
+   ``7804375fe5e4c6d5afe067bd314c42fe0b7d005a1300019c73154dd939da4dda:Splice.Api.FeaturedAppRightV1:FeaturedAppRight``
+   for ``Splice.Api.FeaturedAppRightV1``. The command ``daml damlc
+   inspect-dar`` can be used to find this.
+
+   2. Query the ledger using this ID to retrieve contracts from the Daml
+   ledger that implement the
+   `FeaturedAppRight <https://docs.dev.sync.global/app_dev/api/splice-api-featured-app-v1/Splice-Api-FeaturedAppRightV1.html#type-splice-api-featuredapprightv1-featuredappright-34177>`__
+   interface. See using
+   `toInterface <https://docs.digitalasset.com/build/3.3/reference/daml/interfaces.html#tointerface>`__
+   or
+   `toInterfaceContractId <https://docs.digitalasset.com/build/3.3/reference/daml/interfaces.html#tointerfacecontractid>`__
+   for more information.
+
+   3. In the application, using the ``FeaturedAppRight`` interface, exercise
```

Comment:
Missing: the app's Daml code will have to depend on the `splice-api-featured-app-v1.dar`; and take an argument of type `ContractId FeaturedAppRight` on the choices whose execution should be featured, which allows the choice body to call the `FeaturedAppRight_CreateActivityMarker` choice to create the marker.

Reply:
Added.

Done.